amethyst_animation/src/material.rs

```diff
@@ -8,9 +8,11 @@ use crate::{AnimationSampling, ApplyData, BlendMethod};
 
 /// Sampler primitive for Material animations
 /// Note that material can only ever be animated with `Step`, or a panic will occur.
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+#[serde(untagged)]
```

Comment:
`untagged enum` currently doesn't work for `ron` format ([#117](https://github.com/ron-rs/ron/issues/117))
it works fine with `json` format